py3status/modules/bluetooth.py

```diff
@@ -60,14 +61,23 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 10
-    device_separator = '|'
     format = DEFAULT_FORMAT
     format_device = '{name}'
+    format_separator = '\|'
 
     def post_config_hook(self):
-        # Do deprecation stuff here.  Because of the complexity we can't use
-        # the usual deprecation methods
-        # THIS IS A SPECIAL CASE DO NOT USE AS EXAMPLE CODE.
+        self.format_separator = self.py3.safe_format(self.format_separator)
```

Comment:
I'd prefer this done in the actual `bluetooth()` because maybe it becomes dynamic in future. But I'd also not overwrite `self.format_separator` because that way is pain.  If you do keep it here how about `self._format_separator`